docs/peopleops/offboarding/offboarding-process.md

```diff
@@ -0,0 +1,46 @@
+# Offboarding process
+
+Handover and offboarding of employees is an important process for a number of reasons:
+
+- data security and risk management;
+- efficient organisation and company administration;
+- a chance to get final feedback and review from exiting members; and
+- a opportunity to put the company in a strong position to continue developing projects that run
+  across capstone trimesters.
+
+## Update and correctly store all relevant documents
+
+Ensure that all projects' documentation and processes are up-to-date and have been stored in the
+correct location of the relevant [Thoth Tech repository](https://github.com/thoth-tech) on GitHub.
+Documentation should be clear enough for subsequent members to easily follow, understand and, as
+necessary, continue work on projects.
```

Comment:
Clarity: Might be better to clarify what goes where. Better not to rely on the users.
Documentation to docs repo
Processes to handbook
Code to relevant repo

Reply:
Thanks, @talzid. Done :)